src/MyParcelComApiInterface.php

```diff
@@ -41,11 +41,10 @@ public function authenticate(AuthenticatorInterface $authenticator);
      * Get an array with all the regions, optionally regions can be filtered by
      * country code and region code.
      *
-     * @param string|null $countryCode
-     * @param string|null $regionCode
+     * @param mixed $filters
      * @return CollectionInterface
      */
-    public function getRegions($countryCode = null, $regionCode = null);
+    public function getRegions($filters = null);
```

Comment:
Can you also update the description of the method in the docblock? It might be useful to also mention that a postal code filter is available.